shared/oci.go

```diff
@@ -108,8 +108,8 @@ type ComponentModification func(compvers ocm.ComponentVersionAccess) error
 
 // AddComponentVersionToRepository takes a component description and optional resources. Then pushes that component
 // into the locally forwarded registry.
-func AddComponentVersionToRepository(component Component, repository string, componentModifications ...ComponentModification) error {
-	baseURL := "http://127.0.0.1:5000/" + repository
+func AddComponentVersionToRepository(component Component, repository, scheme string, componentModifications ...ComponentModification) error {
+	baseURL := scheme + "://127.0.0.1:5000/" + repository
```

Comment:
Can you add a validation step that would parse the `baseURL`?

Reply:
+1

Reply:
👍

Reply:
Added a parse. But I'm not checking if the scheme is actually a known scheme. That would be overkill here I believe? :)